packages/components/src/EllipsisLoader.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+const EllipsisLoader = ({
+}) => (
+      <div className="loader">
+        {[...Array(3)].map((n, i) => {
```

Comment:
Since there will always be three dots you can just:
```js
<div className="loader">
  <div className="loader__circle"></div>
  <div className="loader__circle"></div>
  <div className="loader__circle"></div>
</div>
```

Reply:
Also, let's make the loader accessible: 
```diff
const EllipsisLoader = ({
+  label = 'loading'
}) => (
+ <figure className="loader" role="alert" aria-live="assertive" aria-label={label}>
- <div className="loader">
    <div className="loader__circle"></div>
    <div className="loader__circle"></div>
    <div className="loader__circle"></div>
+ </figure>
- </div>
)
```

- `[aria-live]` will tell screen readers to call attention to the element right when it gets added to the dom
- `[aria-label]` labels the element since it doesn't have any text content
- `[aria-role]` tells the screen reader what to interpret the element as